client/wfnews-war/src/main/angular/src/app/components/report-of-fire/photo-page/rof-photo-page.component.ts

```diff
@@ -9,11 +10,63 @@ import { ReportOfFire } from "../reportOfFireModel";
   changeDetection: ChangeDetectionStrategy.OnPush
 })
 export class RoFPhotoPage extends RoFPage {
-  public constructor() {
+  public disableNext: boolean = true;
+  public image : any;
+  captureUrl:any;
+  isCaptured: boolean;
+  images:string[] = [];
+  isFullScreen: boolean = false;
+  public constructor(
+    private changeDetector: ChangeDetectorRef,
+    private el: ElementRef) {
     super()
   }
 
   initialize (data: any, index: number, reportOfFire: ReportOfFire) {
     super.initialize(data, index, reportOfFire)
   }
+
+  async takePhoto(){
+    this.image = await Camera.getPhoto({
+      quality: 100,
```

Comment:
Should any of these hardcoded configs be moved into an external config, so we can adjust them?

Reply:
I dont think we would potentially change them?
quality 100 is the highest value (0 -100) 
allowEditing false is not giving an option for user to crop the photo after taking it. 
Other values are pretty standard.